src/app/app.component.ts

```diff
@@ -7,4 +8,9 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'uofg-book-society';
+  router: Router;
+
+  constructor(private _router: Router){
+    this.router = _router; 
+  }
 }
```

Comment:
These changes just allow us to tell which page we're on (to display the navbar or not).

We inject the router, and check it there.